CONTRIBUTING.md

```diff
@@ -10,25 +10,25 @@ Project maintainers will merge changes that improve the website significantly an
 
 1. Create an issue for your work.
    - You can skip this for trial changes
-   - Reuse an existing issue on the topic, and there is one.
-   - Clearly state that you are going to take on implementingit, if that's the case. You can request that the issue be assigned to you. Note: The issue filer and the implementer don't have be the person.
+   - Reuse an existing issue on the topic, if there is one.
+   - Clearly state that you are going to take on implementing it, if that's the case. You can request that the issue be assigned to you. Note: The issue filer and the implementer don't have be the person.
 2. Create a personal fork of the repository on GitHub (if you don't already have one).
-3. In your fork, create a break off the main (`git checkout -b mybranch`).
+3. In your fork, create a new branch (`git checkout -b mybranch`).
 
-   - Name the branch so that it clearly comminucates your intention, such as `issue-123` or `homepage-issue`.
+   - Name the branch so that it clearly comminucates your intention, such as `yourname/issue-123` or `yourname/homepage-issue`.
    - Branches are useful since they isolate your changes from the upstream. They also enable you to create multiple PRs from the same fork.
 
 4. Make and commit your changes to your brance.
-   - we can create our own commit message guideance.
+   - Commit messages should follow this covention for easy tracking of changes made. `[ISSUE-xxxx] - brief description of changes made`
```

Comment:
This too is more of a suggestion to help keep track of who is making what contribution on what issue.